document/core/text/lexical.rst

```diff
@@ -133,7 +133,7 @@ The *look-ahead* restrictions on the productions for |Tblockchar| disambiguate t
 Annotations
 ~~~~~~~~~~~
 
-An *annotation* is a bracketed token sequence headed by an *annotation id* of the form :math:`\T{@id}`.
+An *annotation* is a bracketed token sequence headed by an *annotation id* of the form :math:`\text{@id}` or :math:`\text{@"..."}`.
```

Comment:
The rendering of `\text{@"..."}` is pretty ugly. Can we use `\ldots` in there to make it prettier?